docs/user-guide/builtins.md

```diff
@@ -193,6 +193,20 @@ response = await fetch("https://example.com", method="POST", body="HELLO").text(
     bug). However, you could use a pass-through proxy service to get around
     this limitation (i.e. the proxy service makes the call on your behalf).
 
+### `pyscript.ffi.to_js`
+
+A utility function to convert Python references into their JavaScript
+equivalents. For example, a Python dictionary is converted into a JavaScript
+object literal (rather than a JavaScript `Map`).
```

Comment:
... unless a `dict_converter` is explicitly specified and the runtime is Pyodide.